test-requirements.txt

```diff
@@ -4,10 +4,6 @@
 #
 #    pip-compile test-requirements.in
 #
-aiohttp==3.9.1
-    # via black
```

Comment:
Surprised that black used to require aiohttp!

Reply:
I was slightly concerned when I saw a bunch of dependencies no longer required and looked into it myself as well, and apparently something with hatchling had a bug somewhere and had extra dependencies counting towards default installation dependencies or something around those lines, and the new release of black resolves those issues. ("extra" dependencies I am referring to (they probably have a proper name but I have forgotten it) are that feature where you can specify your package like `idlemypyextension[user]` instead of just `idlemypyextension` and the [user] one will install more dependencies)